tests/helper_methods_test.py

```diff
@@ -42,3 +42,21 @@ def test_config_week_B_returns_right_option_when_week_is_2(self, mock_date):
         self.assertEqual(week_number, 2)
         self.assertNotEqual(week_number, 1)
         self.assertNotEqual(week_number, 3)
+
+    @patch.dict(Config.config_dict, {'BREAKFAST_TIME': '07:45:00'})
+    @patch.dict(Config.config_dict, {'LUNCHTIME': '13:30:00'})
+    def test_config_get_meal_time_for_breakfast(self, *args):
+        after_meal = Helper.check_rating_time('breakfast', '07:45:01')
+        before_meal = Helper.check_rating_time('breakfast', '07:44:59')
+
+        self.assertTrue(after_meal)
+        self.assertFalse(before_meal)
+
+    @patch.dict(Config.config_dict, {'BREAKFAST_TIME': '07:45:00'})
+    @patch.dict(Config.config_dict, {'LUNCHTIME': '13:30:00'})
+    def test_config_get_meal_time_for_lunch(self, *args):
+        after_meal = Helper.check_rating_time('lunch', '13:30:01')
+        before_meal = Helper.check_rating_time('lunch', '13:29:59')
+
+        self.assertTrue(after_meal)
+        self.assertFalse(before_meal)
```

Comment:
@andela-ydaniju I ran these tests without having the meal settings in my config file and they all passed. This means that these tests here are not really ensuring that we have the right time set. What you are doing here is patching and checking the same patch. What you should check is that the function returns the right template and that it works fine in the functions where it is implemented. I will explain better on slack.